troupon/static/scss/intlTelInput.scss

```diff
@@ -0,0 +1,281 @@
+/**
+ * Variables declared here can be overridden by consuming applications, with
+ * the help of the `!default` flag.
+ *
+ * @example
+ *     // overriding $hoverColor
+ *     $hoverColor: rgba(red, 0.05);
+ *
+ *     // overriding image path
+ *     $flagsImagePath: "images/";
+ *
+ *     // import the scss file after the overrides
+ *     @import "bower_component/intl-tel-input/src/css/intlTelInput";
+ */
+
+// rgba is needed for the selected flag hover state to blend in with
+// the border-highlighting some browsers give the input on focus
+$hoverColor: rgba(0, 0, 0, 0.05) !default;
+$greyText: #999 !default;
+$greyBorder: #CCC !default;
+
+$flagHeight: 15px !default;
+$flagWidth: 20px !default;
+$flagPadding: 8px !default;
+// this border width is used for the popup and divider, but it is also
+// assumed to be the border width of the input, which we do not control
+$borderWidth: 1px !default;
+
+$arrowHeight: 4px !default;
+$arrowWidth: 6px !default;
+$triangleBorder: 3px !default;
+$arrowPadding: 4px !default;
+$arrowColor: #555 !default;
+
+$inputPadding: 6px !default;
+$selectedFlagWidth: $flagWidth + $flagPadding + $arrowWidth + (2 * $arrowPadding) !default;
+
+// image related variables
+$flagsImagePath: "../img/" !default;
+$flagsImageName: "flags" !default;
+$flagsImageExtension: "png" !default;
+
+.intl-tel-input {
+  // need position on the container so the selected flag can be
+  // absolutely positioned over the input
+  position: relative;
+  // keep the input's default inline properties
+  display: inline-block;
+
+  // paul irish says this is ok
+  // http://www.paulirish.com/2012/box-sizing-border-box-ftw/
+  * {
+    box-sizing: border-box;
+    -moz-box-sizing: border-box;
+  }
+
+  .hide {
+    display: none;
+  }
+  // need this during init, to get the height of the dropdown
+  .v-hide {
+    visibility: hidden;
+  }
+
+  // specify types to increase specificity e.g. to override bootstrap v2.3
+  input, input[type=text], input[type=tel] {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Each selector in a comma sequence should be on its own single line